src/Compression/CompressedReadBufferBase.cpp

```diff
@@ -45,6 +51,11 @@ static void validateChecksum(char * data, size_t size, const Checksum expected_c
     if (expected_checksum == calculated_checksum)
         return;
 
+    ProfileEvents::increment(ProfileEvents::CompressedReadBufferChecksumDoesntMatch);
+
+    Stopwatch watch;
+    SCOPE_EXIT({ ProfileEvents::increment(ProfileEvents::CompressedReadBufferChecksumDoesntMatchMicroseconds, watch.elapsedMicroseconds()); });
```

Comment:
you can use ProfileEventTimeIncrement instead

Reply:
Thanks!